### PR TITLE
getting_started/spack.md: provide instructions on updating

### PR DESCRIPTION
**Pull request by harshula**

### CODE REVIEW

File: docs/getting_started/spack.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Set up Spack for building ACCESS models
 
+!!! info
+    Last updated: 30/09/2024
+
```
Comment (atteggiani) [inline docs/getting_started/spack.md:5]:
```suggestion
```

I think the idea of "versioning" the docs make sense, but it's a bigger issue that should involve the whole Hive Docs. 
For a "last update" message, there is already a line (enabled with the [mkdocs-git-revision-date-localized-plugin](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin)) at the bottom of each page on the Hive Docs.

File: docs/getting_started/spack.md [hunk 2]
```diff
@@ -15,19 +18,25 @@ These instructions are tailored specifically for _Gadi_. To use _Spack_ on _Gadi
 
 ## Set up Spack on Gadi
 
+!!! warning
```
Comment (atteggiani) [inline docs/getting_started/spack.md:21]:
```suggestion
!!! tip
```

I would change that to a "tip", but just my preference.

File: docs/getting_started/spack.md [hunk 3]
```diff
@@ -15,19 +18,25 @@ These instructions are tailored specifically for _Gadi_. To use _Spack_ on _Gadi
...
-mkdir <myspack>
-cd <myspack>
+mkdir -p spack/0.22
+cd spack/0.22
 ```
 
 ### Clone the relevant git repositories
 
+!!! info
+    ACCESS-NRI maintains a fork of spack to enable back-porting fixes from more recent spack versions. This fork is the one used in these instructions.
```
Comment (atteggiani) [inline docs/getting_started/spack.md:36]:
```suggestion
    ACCESS-NRI maintains a [fork of spack](https://github.com/ACCESS-NRI/spack) to enable back-porting fixes from more recent spack versions. This fork is the one used in these instructions.
```

**Review by atteggiani — CHANGES_REQUESTED** [review]
Thank you @harshula for the PR.

Looks pretty good, just a few minor comments.

File: docs/getting_started/spack.md (hunk 1 above)
Comment (harshula) [reply docs/getting_started/spack.md:5]: Thanks! I'll remove this.

File: docs/getting_started/spack.md (hunk 2 above)
Comment (harshula) [reply docs/getting_started/spack.md:21]: Will do.

File: docs/getting_started/spack.md (hunk 3 above)
Comment (harshula) [reply docs/getting_started/spack.md:36]: Will do

**Review by atteggiani — APPROVED** [review]
Looks good. 

Thanks @harshula!

**Pull request merged**